src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -3719,8 +3719,8 @@ Bailout:
                 If arguments.Length <> maxCount Then
                     Return New QuickApplicabilityInfo(candidate, CandidateAnalysisResultState.ArgumentCountMismatch, True, False)
                 End If
-            ElseIf arguments.Length < requiredCount OrElse
-               (Not hasParamArray AndAlso arguments.Length > maxCount) Then
+
+            ElseIf arguments.Length < requiredCount OrElse (Not hasParamArray AndAlso arguments.Length > maxCount) Then
```

Comment:
>ElseIf arguments.Length < requiredCount OrElse (Not hasParamArray AndAlso arguments.Length > maxCount) Then [](start = 12, length = 107)

It looks like all changes in this file are stylistic changes, please revert this file to its original state.